package.json

```diff
@@ -1,65 +1,78 @@
 {
   "name": "ember-responsive",
-  "version": "5.0.0",
+  "version": "5.1.0",
   "description": "An ember-cli addon that gives you a simple, Ember-aware way of dealing with media queries.",
   "directories": {
     "doc": "doc",
     "test": "tests"
   },
   "repository": "https://github.com/freshbooks/ember-responsive",
-  "engines": {
-    "node": "10.* || >= 12"
-  },
   "author": "FreshBooks",
   "license": "MIT",
   "scripts": {
     "build": "ember build --environment=production",
-    "lint:js": "eslint .",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
+    "lint:hbs": "ember-template-lint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
-    "test": "ember test",
-    "test:all": "ember try:each"
+    "test": "npm-run-all lint test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each",
+    "prepack": "ember ts:precompile",
+    "postpack": "ember ts:clean"
   },
   "dependencies": {
-    "ember-cli-babel": "^7.26.11"
+    "ember-cli-babel": "^7.26.11",
+    "ember-cli-htmlbars": "^6.1.0"
```

Comment:
Does htmlbars need to be in here or can it be moved to devDependencies? This will affect users of ember-responsive. I think we only need it for testing.

Reply:
Not sure, but I can test it... This was the default setup from `ember addon`

Reply:
Most addons would need it but we don't have any templates

Reply:
Yeah, looks like we don't need htmlbars there, but `tracked-built-ins` does actually need to be